### PR TITLE
Ft/landing

### PR DESCRIPTION
**Pull request by RahmanC**

### CODE REVIEW

File: frontend/tailwind.config.ts
Comment (michojekunle) [inline frontend/tailwind.config.ts:1]: the percentage added in the string can be updated to just numbers alone

File: frontend/src/components/common/SearchInput.tsx
Comment (michojekunle) [inline frontend/src/components/common/SearchInput.tsx:1]: For the Icons can we stick to lucide-react icons, it's quite intuitive and easy to use

File: frontend/src/components/Features.tsx
Comment (michojekunle) [inline frontend/src/components/Features.tsx:1]: Same here for lucide-react icons

File: frontend/src/components/common/Button.tsx
Comment (michojekunle) [inline frontend/src/components/common/Button.tsx:1]: For components let's use shadcn/ui so we don't have to write our components from scratch

**Review by michojekunle — CHANGES_REQUESTED** [review]
Please Look throught the comments on each changes 
- pertaining to packages to use "lucide-react" for icon library and "shadcn-ui" for component library 
- and a few touches on the tailwind-css config

**Comment by RahmanC** [comment]
I've updated the packages and the component.

**Review by michojekunle — APPROVED** [review]
Awesome Stuff Rhammy 🚀🔥

**Pull request merged**